### PR TITLE
Sutherland viscosity model

### PR DESCRIPTION
**Pull request by pgkirsch**
Related to Trello card #51 (https://trello.com/c/1sp9umbV)

### CODE REVIEW

**Comment by pgkirsch** [comment]
@whoburg Ready for review

File: sutherland.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        mu = Variable("mu", "kg/(m*s)", "Dynamic viscosity")
+        T  = Variable("T", "K", "Temperature")
+
+        # Constants
+        T_S   = Variable("T_S", 110.4, "K", "Sutherland Temperature")
+        C_1 = Variable("C_1", 1.458E-6, "kg/(m*s*K^0.5)",
+                       "Sutherland coefficient")
+
+        objective = 1/mu
+        constraints = [(1 + (T_S/T)) * mu <= C_1 * T**0.5]
```
Comment (whoburg) [inline sutherland.py]: cool, nice work - but I'm thinking maybe this needs to default to being a Signomial constraint (flip the inequality).

Reasoning is that many (most?) aerodynamic models will tend to exert downward pressure on mu -- less viscosity = less viscous drag. I know that's a simplistic view of things, but any disagreement?

Comment (pgkirsch) [inline sutherland.py]: True. The GP form works for the Reynolds number matching D8 model though, hence why I left it as is for now. Let me add an SP version.

**Comment by pgkirsch** [comment]
All of these commits are included in PR #45

**Pull request closed without merging**